ci/triage_build.sh

```diff
@@ -18,7 +18,7 @@ fi
 MSG="$(git show -s --format=%s $SHA)"
 KIND="$RUNNER_OS $ARCH"
 
-CIBW_SKIP="pp* *musllinux*"
+CIBW_SKIP="*musllinux*"
```

Comment:
pypy wheels are not enabled by default anymore, and skipping a build that wasn't otherwise enable raises a warning